__test__/lib/Slack_web_api.test.js

```diff
@@ -253,41 +268,41 @@ describe('api methods', () => {
     describe('special cases', () => {
 
         test('chat.postMessage stringifies attachments', () => {
-            instance.chat.postMessage({attachments: []}, cb);
-            expect(instance.callAPI).toHaveBeenCalledWith('chat.postMessage', {attachments: '[]'}, cb);
+            instance.chat.postMessage({ attachments: [] }, cb);
```

Comment:
ew, sorry about all the whitespace changes. I think those are due to fixing linting errors.